app.py

```diff
@@ -18,10 +18,12 @@
 LANGS = dict(zip(constants.get('GH_REPOS').split(','), constants.get('LANGS').split(';')))
 CURRENT = dict(zip(constants.get('GH_REPOS').split(','), constants.get('CURRENT').split(';')))
 
-try:
-  s3 = S3(constants.get('AWS_ACCESS_KEY'), constants.get('AWS_SECRET_KEY'), constants.get('AWS_BUCKET'))
-except:
-  s3 = None
+s3 = None
+if constants.get('GH_COMMENT').lower() == 'true':
```

Comment:
ok! makes sense. the one reason i tied them together was to avoid the red 'Your S3 keys are invalid!' error message, since we intentionally aren't using S3. should i add another constant for that? open to other ideas!